src/mame/igs/goldstar.cpp

```diff
@@ -6933,20 +6933,20 @@ static INPUT_PORTS_START( nfb96 )
 
 	PORT_START("DSW4")
 	PORT_DIPNAME( 0x01, 0x01, "Check Account" )                         PORT_DIPLOCATION("DSW4:1")  // OK
-	PORT_DIPSETTING(    0x00, DEF_STR( No ) )
-	PORT_DIPSETTING(    0x01, DEF_STR( Yes ) )
+	PORT_DIPSETTING(    0x00, DEF_STR( Yes ) )
```

Comment:
Please put No as the first option in order and Yes at the second. Most of MAME is like this and users usually expect to have to go left for no and right for yes.